src/Scrapers/Pipeline/Strategy/FetchStrategy.ts

```
@@ -5,11 +5,24 @@
+  readonly extraHeaders: Record<string, string>;
+}
+
+/** Default fetch options — no extra headers. */
+const DEFAULT_FETCH_OPTS: IFetchOpts = { extraHeaders: {} };
+
 /** Fetch strategy interface — all fetches return strong-typed Procedure. */
 interface IFetchStrategy {
-  fetchPost<T>(url: string, data: Record<string, string>): Promise<Procedure<T>>;
-  fetchGet<T>(url: string): Promise<Procedure<T>>;
+  /** POST with optional extra headers. */
+  fetchPost<T>(url: string, data: Record<string, string>, opts: IFetchOpts): Promise<Procedure<T>>;
```

ESLint

[error] 'url' is defined but never used.

(no-unused-vars)